### ISSUE TITLE
Problem with db.kegg.getId in Database.R

### ISSUE THREAD

**Issue by nalden01** [c-ruttkies/MetFragR#11]
In line 122 of Database.R (part of the db.kegg.getQuery function), the seek variable is overwritten to be a list containing $str and $idx. Then on line 124 and 129 when the program attempts to access seek$mass or seek$mw, it is unable to find them because seek has been overwritten. Therefore, it never enters db.lib.calcmass to calculate the range and the url call to the kegg rest API only contains the originally input mass or molecular weight.

A workaround is to calculate the range manually and perform a call where you specify the exact mass as a range:
strP = "111.0105-111.0771"
params <- List(mass=strP)

**Comment by c-ruttkies** [comment]
Hi,

sorry for the long latency.

We have redesigned the package now and finally upgraded it to version 2.0. This means that all database functions have been removed. The database search and candidate retrieval is included in the function call of e.g. run.metfrag (see the example).

In case of further problems do not hesitate to ask. We will also check the issue messages frequently now.

Kind regards,
Christoph
